specs/core/0_beacon-chain.md

```diff
@@ -826,23 +821,14 @@ def min_empty_validator_index(validators: List[ValidatorRecord], current_slot: i
             return i
     return None
 
-
-def get_fork_version(fork_data: ForkData,
-                     slot: int) -> int:
-    if slot < fork_data.fork_slot_number:
-        return fork_data.pre_fork_version
-    else:
-        return fork_data.post_fork_version
-
-
 def get_domain(fork_data: ForkData,
                slot: int,
                base_domain: int) -> int:
-    return get_fork_version(
-        fork_data,
-        slot
-    ) * 2**32 + base_domain
-
+    if slot < fork_data.fork_slot:
+        fork_version = fork_data.pre_fork_version
+    else:
+        fork_version = fork_data.post_fork_version
+    return fork_version * 2**32 + base_domain
```

Comment:
Not sure I like this simplification. There are use cases for `get_fork_version` that do not have to do with `get_domain` and it IMO adds clarify to specify it explicitly, as it makes it super-clear to understand how to determine whether or not some hard fork has activated.